.github/workflows/pr-test.yml

```diff
@@ -0,0 +1,24 @@
+name: PR-Test
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-test-and-release:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0 # fetch tags
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.20'
```

Comment:
let's include the `release` action update to 1.20 to this PR.

Reply:
which release of go 1.20 do you want?

Reply:
latest 1.20

Reply:
@blewater  you mean this one? goreleaser/goreleaser-action@master

Reply:
or you mean to make release.yml to run as well when open PR?

Reply:
release.yml to have its Go version also updated to 1.20. But that's ok I can do that. Let's wrap up this one.

Reply:
ok, so this is done? I saw that it is already on 1.20

Reply:
Yeah, it's done. Thanks.